.github/workflows/_nodejs.condo.core.tests.yml

```diff
@@ -17,24 +17,37 @@ jobs:
     runs-on: ${{ inputs.runs-on }}
 
     steps:
+      - name: install deps
+        run: |
+          sudo apt update && sudo apt install -y git
+
+      - name: setup docker compose
+        uses: KengoTODA/actions-setup-docker-compose@v1
+        with:
+          version: '2.27.1'
+
       - name: login to sbercloud registry
         uses: docker/login-action@v3
         with:
           registry: ${{ secrets.DOCKER_REGISTRY }}
           username: ${{ secrets.SBERCLOUD_CR_USERNAME }}
           password: ${{ secrets.SBERCLOUD_CR_PASSWORD }}
 
-      - name: run postgresql for tests
-        run: |
-          docker run -e POSTGRES_USER=$POSTGRES_USER -e POSTGRES_PASSWORD=$POSTGRES_PASSWORD -e POSTGRES_DB=$POSTGRES_DB -p="127.0.0.1:5432:5432" -d ${{ secrets.DOCKER_REGISTRY }}/doma/utils/postgres:13.2
-        env:
-          POSTGRES_USER: postgres
-          POSTGRES_PASSWORD: postgres
-          POSTGRES_DB: main
+      - name: checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          ssh-key: ${{ secrets.SSH_DOCK_SERVER_PRIVATE_KEY }}
+          ref: ${{ github.event.pull_request.head.sha || github.ref }}
 
-      - name: run redis for tests
+      - name: run dbs
         run: |
-          docker run -p="127.0.0.1:6379:6379" -d ${{ secrets.DOCKER_REGISTRY }}/doma/utils/redis:6.2
+          cat << EOF > .env
+          POSTGRES_IMAGE=${{ secrets.DOCKER_REGISTRY }}/doma/utils/postgres:13.2
```

Comment:
Can we pass only registry here? (Don't like hard-coded versions here)

Reply:
Here you can pass whatever you want) Not only hard-coded version. By default the image described in the docker-compose file will be used